neurodegeneration/processing-containers/brainvisualize/files/vtkBrainVisual.py

```diff
@@ -473,6 +419,11 @@ def setup_vtk_pipeline(roi,allz_num,out):
 			setup_camera(viewport_keys[i],ren)
 		elif(viewport_keys[i] == ORIENTATION.COLORBAR): #bottom row, last renderer for colorbar
 			add_scalarbar(ren)
+			ren.ResetCamera()
+			camera =  ren.GetActiveCamera()
+			camera.OrthogonalizeViewUp()
+			ren.ResetCameraClippingRange()	
+			#render_scalarbar(relabelMap,need2relabel)
```

Comment:
are these needed here? Does it not work without these lines? If they are needed, they should be moved to the function add_scalarbar()

Reply:
done